nautobot/dcim/api/serializers.py

```diff
@@ -1069,6 +1072,16 @@ class Meta:
 
     def validate(self, data):
 
+        # set interface status to active if status not provided
+        if not data.get("status"):
+            # status is currently required in the Interface model but not required in api_version < 1.3 serializers
+            # which raises an error when validating except status is explicitly set here
+            query = Status.objects.get_for_model(Interface)
+            try:
+                data["status"] = query.get(slug=InterfaceStatusChoices.STATUS_ACTIVE)
```

Comment:
To ensure correctness, we should probably also filter on `content_types` - if someone edited the Active status  so that it's no longer applicable to Interface records, we don't want to create invalid data here. Same comment on other places where you have this logic.

Reply:
Not needed since `Status.objects.get_for_model(Interface)` takes care of that